tests/launch/decisions.cpp

```diff
@@ -0,0 +1,8 @@
+void test_launch() {
+  Points points{Coordinate(1, 1), Coordinate(2, 2)};
+  Parameters parameters{.LENGTH1 = 1};
+  std::vector<bool (*)(Points, Parameters)> lics;
+  lics.push_back(lic0);
+}
```

Comment:
Is this test done? What does/is it supposed to do?

Reply:
Also, I don't believe the `tests/launch/` directory is used in the Makefile, so this test is currently not being run by `make test`.

Reply:
I removed the file completely